### PR TITLE
Tiny doc fixes

### PR DESCRIPTION
**Pull request by rgugliel-da**
### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: docs/source/daml/intro/4_Transformations.rst [hunk 1]
```diff
@@ -64,7 +64,7 @@ You exercise choices using the ``exercise`` function, which takes a ``ContractId
 
 ``exerciseCmd`` returns a ``Commands r`` where ``r`` is the return
 type specified on the choice, allowing the new ``ContractId Contact``
-to be stored in the variable ``new_contactCid``. Just like for
+to be stored in the variable ``newContactCid``. Just like for
```
Comment (cocreature) [inline docs/source/daml/intro/4_Transformations.rst:67]:
```suggestion
to be stored in the variable ``newContractCid``. Just like for
```
Maybe needs other changes to sync up the name.

**Review by cocreature — APPROVED** [review]
:shipit:

File: docs/source/daml/intro/4_Transformations.rst (hunk 1 above)
Comment (rgugliel-da) [reply docs/source/daml/intro/4_Transformations.rst:67]: It's really contacts: https://github.com/digital-asset/daml/blob/main/docs/source/daml/intro/daml/daml-intro-4/daml/Contact.daml#L68

Comment (cocreature) [reply docs/source/daml/intro/4_Transformations.rst:67]: lol I clearly stared at too much daml if I didn’t realize that "contact" is a valid word :)

**Pull request merged**